internal/pkg/ecs/ecs.go

```diff
@@ -47,15 +55,17 @@ type ServiceDesc struct {
 
 // Client retrieves Copilot information from ECS endpoint.
 type Client struct {
-	rgGetter  resourceGetter
-	ecsClient ecsClient
+	rgGetter       resourceGetter
+	ecsClient      ecsClient
+	StepFuncClient stepFunctionsClient
```

Comment:
Does this client need to be exported?

Reply:
They don't since they will be initialized by `New(sess)` so that the caller do not need to initialize these fields individually across package.

Although I think we could probably refactor this such that the clients are initialized from the caller (seems like a better approach to me) - in which case they will need to be exported.